microk8s-resources/default-args/cluster-agent-env

```diff
@@ -0,0 +1,5 @@
+# In a node where FIPS is enabled, ie /proc/sys/crypto/fips_enabled is set to 1
+# Force FIPS to be used with. If not set GOFIPS is implicetely selected.
+export GOFIPS=0
```

Comment:
Should this be commented as well?

Reply:
No. The idea here is that even in a fips capable system we need to start normally (ignore FIPS) and then if the user needs to use FIPS she would need to point MicroK8s to the right libs.